data/json/items/chemicals_and_resources.json

```diff
@@ -833,6 +833,54 @@
     "stack_size" : 100
    },
    {
+    "id": "compost_unf",
+    "type": "AMMO",
+    "category" : "spare_parts",
+    "symbol": "%",
+    "color": "black",
+    "name": "compost",
+    "description": "A reeking mush of rotting vegetation and soil. Activate to start composting it!",
+    "price": 0,
+    "material": "powder",
+    "flags": ["NO_SALVAGE"],
+	"ammo_type" : "components",
+    "weight": 400,
+    "volume": 1,
+	"charges" : 5,	
+    "to_hit": -2,
+    "use_action": {
+        "type": "delayed_transform",
+        "transform_age": 14400, "//": "1 day",
+        "not_ready_msg": "The compost isn't done yet.",
+        "msg": "You pile up the sludge on the ground to compost.",
+        "moves": 150,
+        "target": "compost"
+    }
+   },
+   {
+    "type" : "COMESTIBLE",
+    "id" : "compost",
+    "name" : "compost fertilizer",
+    "name_plural" : "compost fertilizer",
+    "weight" : 350,
+    "color" : "black",
+    "flags" : ["FERTILIZER"],
+    "use_action" : "PLANTBLECH",
+    "container" : "bag_canvas",
+    "comestible_type" : "FOOD",
+    "symbol" : "~",
+    "quench" : -10,
+    "healthy" : -2,
+    "nutrition" : 5,
+    "description" : "Rich, black compost, perfect for plants!",
```

Comment:
I suggest changing the description to: "Rich, black compost. Perfect for plants!"